eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpClusteredVariantWriterTest.java

```diff
@@ -127,51 +125,21 @@ public void saveDifferentTaxonomies() throws Exception {
         assertEquals(secondClusteredVariant, accessions.get(1).getModel());
     }
 
-    @Test
-    public void failsOnDuplicateVariant() throws Exception {
-        ClusteredVariant clusteredVariant = new ClusteredVariant("assembly", TAXONOMY_1, "contig", START_1,
-                                                                 VARIANT_TYPE, VALIDATED);
-        DbsnpClusteredVariantEntity variant = new DbsnpClusteredVariantEntity(
-                EXPECTED_ACCESSION, hashingFunction.apply(clusteredVariant), clusteredVariant);
-
-        thrown.expect(BulkOperationException.class);
-        try {
-            dbsnpClusteredVariantWriter.write(Arrays.asList(variant, variant));
-        } finally {
-            assertEquals(1, importCounts.getClusteredVariantsWritten());
-        }
-    }
-
-    @Test
-    public void failsOnDuplicateNonIdenticalVariant() throws Exception {
-        ClusteredVariant clusteredVariant = new ClusteredVariant("assembly", TAXONOMY_1, "contig", START_1,
-                                                                 VARIANT_TYPE, false);
-        ClusteredVariant duplicateClusteredVariant = new ClusteredVariant("assembly", TAXONOMY_1, "contig", START_1,
-                                                                 VARIANT_TYPE, true);
-        DbsnpClusteredVariantEntity variant = new DbsnpClusteredVariantEntity(
-                EXPECTED_ACCESSION, hashingFunction.apply(clusteredVariant), clusteredVariant);
-        DbsnpClusteredVariantEntity duplicateVariant = new DbsnpClusteredVariantEntity(
-                EXPECTED_ACCESSION, hashingFunction.apply(duplicateClusteredVariant), duplicateClusteredVariant);
-
-        thrown.expect(BulkOperationException.class);
-        try {
-            dbsnpClusteredVariantWriter.write(Arrays.asList(variant, duplicateVariant));
-        } finally {
-            assertEquals(1, importCounts.getClusteredVariantsWritten());
-        }
-    }
-
     @Test
     public void allNonDuplicatedRecordsWillBeWritten() {
-        // batch of 6 variants, 5 unique and 1 duplicate that is in the middle of the batch
+        // batch of 6 variants, 5 unique and 2 duplicates that are in the middle of the batch
         ClusteredVariant firstClusteredVariant = new ClusteredVariant("assembly", TAXONOMY_1, "contig", START_1,
```

Comment:
Could these variables be renamed to follow our usual naming `clusteredVariant1`, `clusteredVariant2`, etc? It makes easier to read which is which that using the ordinals in text.

Reply:
Solved in test class refactor

Reply:
Done in test class refactor